python/tests/test_regression.py

```diff
@@ -0,0 +1,62 @@
+from vw_executor.vw import Vw
+from vw_executor.vw_opts import Grid
+from numpy.testing import assert_allclose
+import pandas as pd
+import numpy as np
+import random
+
+with open("regression1.txt", "w") as f:
+    for i in range(10000):
+        x = random.uniform(1, 100)
+        y = 5
+        f.write(f"{y} |f x:{x}\n")
+
+
+def core_test(files, grid, outputs, job_assert):
+    vw = Vw(".vw_cache", reset=True)
+    result = vw.train(files, grid, outputs)
+    job_assert(result)
+
+
+def test_regression_weight():
+    options = Grid(
+        {
+            "#base": ["-P 50000 --preserve_performance_counters --save_resume "],
+            "#reg": ["", "--coin", "--ftrl", "--pistol"],
+        }
+    )
+
+    def assert_job(job):
```

Comment:
This code seems quite generic - "compare weights with expected values with given tolerance", but not reusable at the moment.